DigitalLearningSolutions.Web/Controllers/FrameworksController/Frameworks.cs

```diff
@@ -680,11 +680,11 @@ public IActionResult AddCollaborator(string actionname, string userEmail, bool c
                 }
                 else if (collaboratorId == -2)
                 {
-                    TempData["FrameworkError"] = $"User with the email address {userEmail} has been previously added";
+                    TempData["FrameworkError"] = $"User with the email address has been previously added";
                 }
                 else if (collaboratorId == -4)
                 {
-                    TempData["FrameworkError"] = $"The email address {userEmail}  must match registered DLS Admin account";
+                    TempData["FrameworkError"] = $"The email address must match registered DLS Admin account";

```

Comment:
Tweak to wording:

"The email address must match registered **a** DLS Admin account"

Reply:
fixed